src/main.js

```diff
@@ -375,6 +376,23 @@ app.on('ready', function() {
   window_options.title = app.getName();
   mainWindow = new BrowserWindow(window_options);
 
+  if (process.platform === 'darwin') {
+    session.defaultSession.on('will-download', (event, item, webContents) => {
```

Comment:
No, I tried it, but it has no effect. This is currently the only way I could manage to open the save dialog in the correct folder.